src/main/java/seedu/address/model/supplier/Company.java

```diff
@@ -27,8 +28,8 @@ public class Company {
     public Company(String company) {
         requireNonNull(company);
         checkArgument(isValidCompany(company), MESSAGE_CONSTRAINTS);
-        company = normalizeCompanyName(company);
-        this.value = company;
+        this.value = normalizeCompanyName(company);
+        this.displayValue = company;
     }
```

Comment:
Add the picture from the forum as proof this is a bug fix so we can justify easily

Reply:
Done added!